modules/op-creating-repository-cr.adoc

```diff
@@ -0,0 +1,32 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="creating-repository-cr_{context}"]
+= Creating the Repository custom resource
+
+You can use the `tkn pac` CLI or other alternative methods to create a `Repository` custom resource (CR) inside the target namespace. For example:
```

Comment:
The considerations are valid but, again, this is out of scope. The content was already there, but it was in the assembly, see https://docs.openshift.com/container-platform/4.13/cicd/pipelines/using-pipelines-as-code.html#using-repository-crd-with-pipelines-as-code_using-pipelines-as-code . I moved the content from the assembly into a module and gave it a heading but did not modify it otherwise.